cloudfour/cloudfour.com-patterns
# Object: Bunch

## Overview

> A tight cluster of visual elements with a small amount of overlap. Useful for displaying avatars alongside bylines where multiple authors are equally credited.

## Screenshots

<img width="874" alt="Screen Shot 2021-02-03 at 11 42 24 AM" src="https://user-images.githubusercontent.com/69633/106800522-6d5ba100-6615-11eb-8959-26b01ee2a828.png">

## Testing

1. Confirm GitHub checks pass
2. Try out object in [deploy preview](https://deploy-preview-1130--cloudfour-patterns.netlify.app/?path=/docs/objects-bunch--of-avatars)

---

- See #487 #1093
